### ISSUE TITLE
Revisit PDS Deep Archive Integration with Registry

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/pds-deep-archive#115]
Follow-on to #105 , awaiting some updates from PDS API in order to wrap-up the integration.

**Comment by jordanpadams** [comment]
August Status: Deferred to B12.1 due to https://github.com/nasa-pds/pds-deep-archive/issues/107 being blocked by Registry upgrade. Functionality still works well enough for release with this build.
